packages/terra-theme-provider/src/ThemeProvider.jsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import className from 'classnames/bind';
```

Comment:
Does the theme-provider apply any localized styles? I'm not sure we need to use `classnames/bind`.